src/js/components/Anchor/stories/Size.js

```diff
@@ -11,6 +11,9 @@ const SizeAnchor = () => (
           <Anchor size={size} label={size} href="#" />
         </Box>
       ))}
+      <Box margin="small">
```

Comment:
Minor comment, but any reason not to just add `10px` as another index in the array at line 9?

Reply:
good point!